_tech-blog/2020-08-31-gsoc-20-resume.markdown

```diff
@@ -0,0 +1,61 @@
+---
+title: GSoC 20 - Automatized support for Organised Editing Guidelines
+date: 2020-08-31 12:30:00 Z
+Summary Text: Here is the resume from the GSoC' 20 project that adds
+  automatized support for Organised Editing Guidelines in
+  Tasking Manager.
+Feature Image: "/uploads/gsoc-20-architecture.jpg"
+Person:
+- Wille Marcel
+---
+
+
+# GSoC 20 - Automatized support for Organised Editing Guidelines
+
+## The problem
+&nbsp;&nbsp;&nbsp;&nbsp;The OpenStreetMap (OSM) project has established formal [Organised Editing Guidelines](https://wiki.osmfoundation.org/wiki/Organised_Editing_Guidelines), which require all groups and organized activities of mapping on OpenStreetMap to report transparently how they organize and what they are mapping on.
```

Comment:
Remove all `&nbsp;`, let one blank line to break the paragraphs

Reply:
Alright, updated without the `&nbsp` and with a blank line to break the paragraphs.